docs/user_guide/assessment/common_fairness_metrics.rst

```diff
@@ -299,7 +305,128 @@ capture the costs of missclassification disparities.
 
 
 
+.. _assessment_four_fifths:
+
+The Four Fifths Rule: Often Misapplied
+--------------------------------------
+
+In the literature around fairness in machine learning, one will often find
+the so-called "four fifths rule" or "80% rule" used to assess whether a model
+(or mitigation technique) has produced a 'fair' result.
+Typically, the rule is implemented by using the demographic parity ratio introduced
+in the :ref:`assessment_demographic_parity` section above, with a result considered
+'fair' if the ratio exceeds 80% for all identified subgroups.
```

Comment:
Would it be helpful to include a brief example of calculating the four fifths rule here? Understand if we just briefly want to touch on this topic but don't want to go into too much detail!

Reply:
I've provided a link to the function people could use, but I think it would be odd to give a code example of 'how to do this' in a section which is saying "don't do this."